config/settings.yml

```diff
@@ -30,18 +30,23 @@ saml_ssoe:
   response_signing: false
   response_encryption: false
 
+ssoe_eauth_cookie:
```

Comment:
I take it we'll need a PR to update the prod/staging/dev config files in the devops repo?

Reply:
Yeah.  I have those updates ready locally.  Just need to open the PR